manual/sphinx/user_docs/installing.rst

```diff
@@ -465,3 +355,211 @@ If the tests pass, congratulations! You have now got a working
 installation of BOUT++. Unless you want to use some experimental
 features of BOUT++, skip to section [sec-running] to start running the
 code.
+
+Installing BOUT++ (experimental)
+--------------------------------
+
+.. _sec-install-bout:
+
+Most BOUT++ users install and develop their own copies in their home directory,
+so do not need to install BOUT++ to a system directory.
+As of version 4.1 (August 2017), it is possible to install BOUT++ but this is
+not widely used and so should be considered experimental. 
+
+After configuring and compiling BOUT++ as above, BOUT++ can be installed
+to system directories by running as superuser or ``sudo``:
+
+**NOTE:** Do not do this unless you know what you're doing!
+
+.. code-block:: bash
+
+   $ make install
+
+This will install the following files under ``/usr/local/``:
+   
+* ``/usr/local/bin/bout-config``  A script which can be used to query BOUT++ configuration and compile codes with BOUT++.
+
+* ``/usr/local/include/bout++/...`` header files for BOUT++
+    
+* ``/usr/local/lib/libbout++.a``  The main BOUT++ library
+
+* ``/usr/local/lib/libpvode.a`` and ``/usr/local/lib/libpvpre.a``, the PVODE library
+
+* ``/usr/local/share/bout++/pylib/...`` Python analysis routines
+
+* ``/usr/local/share/bout++/idllib/...`` IDL analysis routines
+  
+* ``/usr/local/share/bout++/make.config`` A ``makefile`` configuration, used to compile many BOUT++ examples
+
+
+To install BOUT++ under a different directory, use the ``--prefix=`` flag e.g. to install
+in your home directory:
+
+  .. code-block:: bash
+
+   $ make install --prefix=$HOME/local/
```

Comment:
Good spot, thanks! Now fixed I think. 
I put the instructions on passing to make install first, just because if users have got to that point it seemed odd to ask them to run configure and make again. I'm not sure though how many software packages actually work with passing prefix= to make install, as I agree the "normal" way is through configure.